src/login/logind-session.c

```diff
@@ -1170,7 +1170,7 @@ static int on_bus_track(sd_bus_track *track, void *userdata) {
         return 0;
 }
 
-int session_set_controller(Session *s, const char *sender, bool force) {
+int session_set_controller(Session *s, const char *sender, bool force, bool prepare) {
```

Comment:
not a big fan of functions taking multiple bools (usually a flags param is nicer than as the invocations become more descriptive), but I figure as long as its just two its OK